packages/app-store/BookingPageTagManager.tsx

```diff
@@ -37,12 +37,13 @@ export default function BookingPageTagManager({
           return (
             <Script
               src={parseValue(script.src)}
-              id={`${appId}-${index}`}
```

Comment:
Unrelated change to migration, will revert.

Reply:
Sorry bout that one. Not sure why that got removed 🤔